### DISCUSSION TITLE
authenticates -> scrapes all usernames -> when i choose username it will not scrape

### DISCUSSION THREAD

**Discussion by YESANDNOTHENYESAGAIN** [DIGITALCRIMINALS/OnlyFans#451] [General]
What my title says. As soon as i choose username 240 for example it will scrape the metadata, create folders, get the profile picture, but then will not download anything. In fact it will skip it and start download other random OF accounts. 

Ive tried using both versions. The one you directly download and run as is. And the version where you run the updater and Poetry. Both version do have this weird ass interaction. Any help is appreciated and if you want to buy OF or trade OF for help with this HMU.

**Comment by clouds321** [comment]
I had similar problem in the past. 

what is your setting for jobs scraping????

`"jobs": {
          "scrape": {
            "subscriptions": true,
            "messages": false,
            "paid_content": false}`

this part is really confusing to me, but even if messages and paid are set to false they still get scrapped when you choose a model. 

If they are set to true it will scrape EVERY message and paid content first, try setting them to false. 

Or might be something else IDK...

**Comment by m4rc1nt0sch** [comment]
> if you want to buy OF or trade OF for help with this HMU.

I doubt someone here will help you making a business out of stealing content.
